dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -134,11 +150,44 @@ public static Response truncateUTF8(String s, int maxBytes) {
             }
             return new Response(s, false);
         }
-
+        
+        public byte[] unzipGcsFile(GcsPath zipFile, GcsUtil gcsUtil) throws IOException {
+            byte[] buffer = new byte[1024];
```

Comment:
No there's no good reason. It would make sense if we were writing to an output file buffer to keep the memory consumption to a reasonable amount, but using a byte buffer stores the whole file in memory anyway. 1 KB just happens to be the amount I see used traditionally. (read: just following what other unzip code examples do)

FWIW I guess we could dial this up to 1 MB or something and try to read the file in one or two big gulps. I'll add a TODO to play with the value.